app.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controllers");
+const { badPath } = require("./controllers/app.controllers");
+const app = express();
+app.use(express.json());
```

Comment:
Line 5 is unnecessary as there aren't any POST or PATCH endpoints yet in your api